### PR TITLE
fix vendor dependencies

### PR DESCRIPTION
**Pull request by bassam**
resolve conflict between client-go dependencies and external-provisioner

Addresses #689

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -1,9 +1,37 @@
-hash: 791fe92e4f8d196d1f28a3e4400108969dd94ddd2de0283916fa7c00646aa7a8
-updated: 2017-04-14T12:56:26.659073101-07:00
+hash: 87157469836148176d50b8368601122106cba44d7992b8fb6f2478da1a371ae3
+updated: 2017-06-11T21:49:37.68482446-07:00
 imports:
 - name: bitbucket.org/ww/goautoneg
   version: a547fc61f48d567d5b4ec6f8aee5573d8efce11d
   repo: https://github.com/quantum/goautoneg.git
+- name: github.com/aws/aws-sdk-go
```
Comment (kokhang) [inline glide.lock]: Could you remind me why we need to bring this aws dependency?

File: glide.yaml [hunk 2]
```diff
@@ -21,18 +21,17 @@ import:
...
   version: a547fc61f48d567d5b4ec6f8aee5573d8efce11d
   repo: https://github.com/quantum/goautoneg.git
 - package: github.com/coreos/pkg/capnslog
+- package: github.com/kubernetes-incubator/external-storage
+  version: v2.1.0
 - package: k8s.io/client-go
-  version: release-3.0
-  subpackages:
-  - kubernetes/typed/core/v1
+  version: 76153773eaa3a268131d3d993290a194a1370585
```
Comment (kokhang) [inline glide.yaml]: are these version tags used to match the dependency of external-storage?

File: glide.lock (hunk 1 above)
Comment (bassam) [inline glide.lock]: we don't. external-storage brings in aws-sdk

File: glide.yaml (hunk 2 above)
Comment (bassam) [inline glide.yaml]: yes. from their release notes.

**Pull request merged**